harness/determined/common/api/experiment.py

```diff
@@ -14,6 +14,12 @@
 from determined.common.api import request as req
 
 
+def patch_experiment_v1(master_url: str, exp_id: int, patch_doc: Dict[str, Any]) -> None:
```

Comment:
>  python sdk?

Do you mean the new APIs? Ideally, I'd want us to use the python-sdk that @mh-determined  worked on or directly use the swagger bindings instead of manually writing apis like this. both of those would require bigger changes that I'd like to keep out of this PR. This is following other uses of calls to `/api/v1/X` in our cli code which isn't as ideal as using the generated code or the sdk directly to me but it's a step toward it.

Reply:
> Ideally, I'd want us to use the python-sdk that @mh-determined worked on or directly use the swagger bindings instead of manually writing apis like this

this is exactly what i mean.

> This is following other uses of calls to /api/v1/X in our cli code which isn't as ideal as using the generated code or the SDK directly to me but it's a step toward it.

yeah, i was just curious when you thought the next step should be.

Reply:
could we consider this tech debt and do a tech-debt week on?

Reply:
I added an entry in the sheet